aiida/cmdline/commands/cmd_computer.py

```diff
@@ -141,11 +142,11 @@ def _computer_create_temp_file(transport, scheduler, authinfo):  # pylint: disab
 
     file_content = "Test from 'verdi computer test' on {}".format(datetime.datetime.now().isoformat())
     echo.echo('> Creating a temporary file in the work directory...')
-    echo.echo('  `-> Getting the remote user name...')
+    echo.echo('  -> Getting the remote user name...')
```

Comment:
Maybe it is nice to put the `[OK]` on the same line as the first line.
This would go for all other lines as well where there is no other intermediate result but just an ok.
Put the message with ellipses and OK on the same line

Reply:
I agree this looks nicer and I wanted to do this at first. It's nasty though - what do you do if the test fails?

During test failures, some commands print random stuff, which will then inevitably land on the same line.
For the "echo" statements originating from the command itself one could handle it, but also there you would need to do quite a bit of if/else in order to avoid having stuff printed on the same line that shouldn't be.

I've made the changes here https://github.com/ltalirz/aiida-core/tree/computer-test-echo-nl-false
Let me know what you prefer.

Reply:
If it fails, we can still print the error after a `[FAILED]` right? Or even on the next line if it is multiple lines. But I would imagine something like this:
```
* Getting job list... [OK]: Found 6 jobs in the queue
* Checking that no spurious output is present... [FAILED]
* Getting the remote user name... [OK]: <ltalirz>
* Retrieving the file and checking its content... [FAILED]: file not found
```

Reply:
well, that is if you control everything that happens between printing "..." and the failure.
currently, many of the methods being called here just randomly print stuff (and not just the ones defined inside this file).
I'd have to start capturing all of this - can be done but is quite some work

Reply:
we can just put the calling blocks in `with Capturing()` no? If you are ok with it, I will gladly update your other branch to do it.

Reply:
sure, please go ahead!
you have write acces to my fork already